include/bluetooth/hci.h

```diff
@@ -1825,6 +1825,49 @@ typedef bool bt_hci_vnd_evt_cb_t(struct net_buf_simple *buf);
   */
 int bt_hci_register_vnd_evt_cb(bt_hci_vnd_evt_cb_t cb);
 
+
+/** Allocate a HCI event buffer.
+ *
+ * This function allocates a new buffer for a HCI event. It is given the
+ * OpCode (encoded e.g. using the BT_OP macro) and the total length of the
```

Comment:
BT_OP is used for command opcodes not for event codes. I think it'd be good to disambiguate here and use "event code" events.

Reply:
Right, will fix that.